backend/src/main/java/com/bakdata/conquery/apiv1/frontend/FEValue.java

```diff
@@ -1,33 +1,57 @@
 package com.bakdata.conquery.apiv1.frontend;
 
 import java.util.Comparator;
-import java.util.Map;
+import java.util.List;
+import java.util.Objects;
 
-import lombok.AllArgsConstructor;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.google.common.collect.ImmutableList;
 import lombok.Data;
+import lombok.NonNull;
 import org.jetbrains.annotations.NotNull;
 
 /**
  * This class represents a values of a SELECT filter.
  */
 @Data
-@AllArgsConstructor
 public class FEValue implements Comparable<FEValue> {
-	private static final Comparator<FEValue> COMPARATOR = Comparator.comparing(FEValue::getLabel).thenComparing(FEValue::getValue);
+	private static final Comparator<FEValue> COMPARATOR = Comparator.comparing(FEValue::getValue)
+																	.thenComparing(FEValue::getLabel);
+	@NotNull
+	private final String value;
 
 	@NotNull
 	private final String label;
 
-	@NotNull
-	private final String value;
+	private final String optionValue;
 
-	private Map<String, String> templateValues;
+	@JsonCreator
+	public FEValue(@NonNull String value, @NonNull String label, String optionValue) {
+		this.value = value;
+		this.label = Objects.requireNonNullElse(label, value);
+		this.optionValue = optionValue;
+	}
 
-	private String optionValue;
+	public FEValue(String value, String label) {
+		this(value, label, null);
```

Comment:
Sollte hier der optionValue nicht dem value gesetzt werden? Was macht das Frontend sonst ohne optionValue?

Reply:
optionValue ist das was in der großen Liste angezeigt wird, ich denke sonst wird label genommen.

Reply:
@Kadrian the autocomplete/search endpoints have the fields value, optionValue and label, are all three necessary? The prior code had optionValue as optional so I copied that behaviour.